docs/package-sync-plugin.md

```diff
@@ -201,6 +204,50 @@ gradlew :site.demo:packageSync -Pfilter.path=C:/aem/custom-filter.xml
 gradlew :site.demo:packageSync -Pfilter.roots=[/etc/tags/example,/content/dam/example]
 ```
 
+## Task `packageConfig`
+
+Synchronizes OSGi configuration as XML files put into JCR content.
+
+First of all, changing OSGi configuration by hand using GUI dialog when accessing */system/console/configMgr* is not recommended.
+Instead, configuration changes should be applied by providing OSGi config node in built CRX package under path _/apps/${appName}/config_.
+Such file could be created from scratch, but this is generally speaking error-prone process, because of escaping forbidden characters and delimiters, type casts etc.
+
+To simplify that process, task `packageConfig` could be used. 
+It will download actual OSGi configuration values as XML file and put it at desired path to be later deployed within built CRX package.
+
+Alternatively, customized values could be set on OSGi configuration GUI dialog then task `packageConfig` could be run to synchronize changes and save them in VCS in a proper format.
+However that approach has a little disadvantage. After changing anything by hand on OSGi configuration dialog, changes done in XML files for appropriate OSGi configuration might be longer reflected on dialog after installing CRX package.
+Then, as a workaround, try with deleting configuration using button on dialog then update values in XML and deploy package again to restore binding.
```

Comment:
Those two paragraphs are not clear to me.

Did you mean, that if you would perform changes from configManager and then redeploy package with the same configuration that you changed, version from configManager would get precedence?

Reply:
yep, sometimes. I tried to clarify how it is working but looks like it is working intermittently ;

when always changing values by amending XML files it is working always predictably